### PR TITLE
[xbmc[[cleanup] Add a variadic overload to AddFileToFolder to simplify usage

### PR DESCRIPTION
**Pull request by Paxxi**

### CODE REVIEW

**Comment by Paxxi** [comment]
jenkins build this please

**Comment by Paxxi** [comment]
jenkins build this please

**Comment by Paxxi** [comment]
jenkins build this please

File: xbmc/addons/Repository.cpp [hunk 1]
```diff
@@ -147,7 +147,43 @@ CRepository::FetchStatus CRepository::FetchIfChanged(const CRepository::DirInfo&
     content = std::move(buffer);
   }
 
+<<<<<<< 5a7353c274e125cd840acd4f937c44a33033731c
```
Comment (wsnipex) [inline xbmc/addons/Repository.cpp]: rebase artifacts, also further down

File: xbmc/utils/URIUtils.h [hunk 2]
```diff
@@ -189,7 +189,13 @@ class URIUtils
                                 const std::string& pathInArchive = "",
                                 const std::string& password = "");
 
-  static std::string AddFileToFolder(const std::string &strFolder, const std::string &strFile);
+  static std::string AddFileToFolder(const std::string& strFolder, const std::string &strFile);
+  template <typename... T>
+  static std::string AddFileToFolder(const std::string &strFolder, const std::string&strFile, T... args)
```
Comment (wsnipex) [inline xbmc/utils/URIUtils.h]: missing blank in std::string&strFile

**Comment by Paxxi** [comment]
jenkins build and merge this please

**Pull request merged**